### ISSUE TITLE
Release `5212`: Many artefacts missing

### ISSUE THREAD

**Issue by mara004** [bblanchon/pdfium-binaries#88]
As you can see, the latest release [`PDFium 106.0.5212.0`](https://github.com/bblanchon/pdfium-binaries/releases/tag/chromium%2F5212) lacks artefacts for many platforms (only 9 assets instead of the usual 20).

Shouldn't the release workflow stop if building for one platform fails? Also I'm confused because the workflow run that corresponds to this release does not appear to be listed on the [actions panel](https://github.com/bblanchon/pdfium-binaries/actions), or have I overlooked something?

The scheduled workflow of pypdfium2 currently assumes that all required binaries are present. If this isn't the case, I fear that the repository would be left in a half-broken state, or that some released packages would be invalid.
I'm now away for some time, so I wanted to ask if you think next Monday's release will be complete again, or if I should better disable the schedule for now?

Thanks!

**Comment by bblanchon** [comment]
Hi @mara004,

Thank you for reporting this issue.

I had a tough time getting this build because of two outages: one from chromium.googlesource.com and another from musl.cc.
I had to rerun jobs multiple times, and for some reason, it created two releases with the same tag instead of gathering all the artifacts together.
I manually fixed the release, but it seems one artifact is still missing.

I'll start a new build today; hopefully, it will work.

Best regards,
Benoit
